apps/src/lib/client/rpc.rs

```diff
@@ -340,8 +340,7 @@ pub async fn query_transparent_balance<'a>(
                             balance
                         );
                     }
-                    Err(e) => {
-                        display_line!(context.io(), "Querying error: {e}");
+                    Err(_) => {
```

Comment:
I think this error should be suppressed only when it's `QueryError::NoSuchKey` - any other errors should be shown instead of "No balance found" print

Reply:
All build tests passed

Reply:
@tzemanovic can you my changes? 

Thank you!